ui-components/autocompleter.md

```diff
@@ -8,7 +8,7 @@ keywords: autocomplete
 
 ## Overview
 
-An `autocompleter` enables users to quickly find and select from a pre-populated list of values as they type, leveraging searching and filtering. It provides suggestions to insert while the user is typing into the content. For example, with the [charmap]({{site.baseurl}}/plugins/opensource/charmap/) plugin enabled, typing **:amp** should show the ampersand item in the menu. To activate the `autocompleter`, the user must precede the trigger character (colon in this case) with a separator character, such as a space or a newline. Pressing `esc` should close the autocomplete menu.
+An autocompleter displays suggestions while the user is typing. Suggestions are shown when the trigger character is typed in (such as '`:`'). Pressing the Escape key should close the autocompleter.
```

Comment:
We've lost this bit of data in this rewrite, can you please make sure that remains as it's important:
> the user must precede the trigger character (colon in this case) with a separator character, such as a space or a newline.

Reply:
```suggestion
An autocompleter displays suggestions while the user is typing. Suggestions are shown when the trigger character is entered after a space or at the start of a new line (such as '` :`'). Pressing the Escape key should close the autocompleter.
```